test-clients/src/main/java/com/hedera/services/bdd/suites/contract/precompile/WipeTokenAccountPrecompileSuite.java

```diff
@@ -216,7 +221,7 @@ private HapiApiSpec wipeNonFungibleTokenScenarios() {
                                                             asAddress(vanillaTokenID.get()),
                                                             asAddress(accountID.get()),
                                                             List.of(2L))
-                                                    .payingWith(ACCOUNT)
+                                                    .payingWith(ADMIN_ACCOUNT)
                                                     .via(
                                                             "wipeNonFungibleAccountDoesNotOwnTheSerialTxn")
```

Comment:
Similarly to the test for fungible token, you can add a test case where you try to wipe the token with the `ADMIN_ACCOUNT` first without having a `WIPE_KEY` (you can remove it in the account creation). Then update the `ADMIN_ACCOUNT` with the `WIPE_KEY` and then execute all of the test cases starting from here ('wipeNonFungibleAccountDoesNotOwnTheSerialTxn') to the bottom. 

Basically you can plug a new test case at the very beginning and then update the account with the key.